src/apps/frontend/utils/use-timer.hook.ts

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+
+import { RESEND_DELAY_IN_MILLISECONDS, MILLISECONDS_IN_A_SECOND } from '../constants/timer';
```

Comment:
This is not right. Everyone may want to use timer hook with different delays. I believe you want to accept this as a property to the hook.